conda/recipes/cudf/meta.yaml

```diff
@@ -92,6 +94,8 @@ requirements:
     - cubinlinker  # CUDA enhanced compatibility.
     - cuda-python >=11.7.1,<12.0a0
     {% else %}
+    - cuda-cudart
+    - libcufile  # [linux64]
```

Comment:
If we made the change above, no changes would be needed here. Mentioning just to be explict